### PR TITLE
Deliverable 1 and 2

### PR DESCRIPTION
**Pull request by 44Silver4**
I have updated both Deliverable 1 and 2 for submission.

### CODE REVIEW

File: 1907-03-11.xml [hunk 1]
```diff
@@ -0,0 +1,1824 @@
...
+                            Alexandretta, Mersina, <date when="1907-02-18">Febr. 18</date>; <date when="1907-03-04">March 4</date>, <date when="1907-03-18">18</date> ; <date when="1907-04-01">April 1</date>. <date when="1907-04-15">15</date>, <date when="1904-04-29">29</date> ; <date when="1907-05-13">May 13</date>, <date when="1905-05-27">27</date> ; <date when="1907-06-10">June 10</date>, <date when="1907-06-24">24</date>.</p>
+                        <p><hi rend="bold">Syrian-Cyprus Line.</hi> Beyrout, Limassol, Larnaca, Mersina,
+                            <date when="1907-02-26">Febr. 26</date>; <date when="1907-03-12">March
+                                12</date>, <date when="1907-03-26">26</date> ; <date when="1907-04-09">April
+                                    9</date>. <date when="1907-04-23">23</date> ; <date when="1907-05-07">May
+                                        7</date>, <date when="1905-05-21">21</date> ; <date when="1907-06-04">June
+                                            4</date>, <date when="1907-06-18">18</date>.</p>
+                        <p><hi rend="
```
Comment (huoyanzhizhou) [inline 1907-03-11.xml]: Here you close the page division too early, thus the rest of this page no longer belongs to page 2. Please check your outline window and fix this.

Comment (huoyanzhizhou) [reply 1907-03-11.xml]: It seems that my comment is not shown in the accurate line. Please check you line 1697 and your outline window to see.

**Comment by whanley** [comment]
Please make sure that all of your work is in the file named "1907-03-11.xml", then delete the file named "1907-03-11-p1.xml" from your fork of the content repository and send a new pull request. We'll look at your work then.

**Comment by whanley** [comment]
You're getting there! A few things:
- It looks like you still have some content to update on page 1. Also, the when= attribute of your `<date>` tags needs to be updated to match your page's text. Find the problems using this XPath query: `//date/@when`
- line 1697 contains a `</div>` that closes page 2 prematurely. Move this to the end of the page.
- There are templates for the two items that follow
- The article staring at line 1744 ("The Suez Canal") is its own div.

**Comment by whanley** [comment]
Course ended without completion.

**Pull request closed without merging**